micromamba/etc/profile.d/mamba.sh.in

```diff
@@ -1,6 +1,6 @@
 echo "WARNING: @CMAKE_INSTALL_PREFIX@/etc/profile.d/mamba.sh (the file emitting this warning) is deprecated."
-echo "WARNING: This file will be removed in mamba 2.1.0."
-echo "WARNING: Please use 'mamba shell init' to get the correct shell scripts for your shell."
+echo "WARNING: This file will be removed in mamba 2.3.0."
```

Comment:
Note to reviewers: are we OK extending the deprecation period of this file to 2.3.0?

Reply:
How does this look to you?
```suggestion
echo "WARNING: This file will be removed on the release on the next release after 30th September 2025."
```

Reply:
Done with 53d503c.